internal/testrunner/testrunner.go

```diff
@@ -1,9 +1,21 @@
 package testrunner
 
 import (
+	"database/sql"
+
 	"github.com/dogmatiq/example"
 	"github.com/dogmatiq/testkit"
 )
 
-// Runner is a test runner for the example app.
+// Runner is a test runner for the example application, without projections.
 var Runner = testkit.New(&example.App{})
```

Comment:
I'd like to get rid of this variable now that there is a `New()` function, and instead update `New()` and `example.NewApp()` to accept a `nil` value for `db`, in which case it doesn't initialize the projection handlers.

Reply:
As discussed, this will be done in a separate PR. I created issue #36